### ISSUE TITLE
POS multiple punctuation issues

### ISSUE THREAD

**Issue by abdmob** [stanfordnlp/CoreNLP#758]
I have some issues with the sentences with the "!!!" or "???" at the end. CoreNLP POS produces a bit illogical results for them:

For example, for `Hello, sir!!!` I'm receiving the latest POS tag for one `!!!` token  as: 

> NN (Noun, singular or mass)

I was expecting something like `.`,`!` or SYM token (or tokens) like Stanford NLP do with one `!`).

To compare behaviour, I also have tested this example with Google NLP and received (more logical) 3 PUNCT tokens (instead one token).

Does anybody know any tricks or settings to change this behaviour?

I'm using Stanford NLP v3.9.1 and my settings are:

```
      "language", "english", 
      "tokenize.class", "PTBTokenizer",
      "tokenize.language", "en",
      "pos.class", "edu.stanford.nlp.tagger.maxent.MaxentTagger",
      "pos.model", "edu/stanford/nlp/models/pos-tagger/english-left3words/english-left3words-distsim.tagger"
```

**Comment by abdmob** [comment]
I'm not justifying using this "kind of English", you know. I'm just trying to parse a modern twitter language :)

**Comment by manning** [comment]
I agree this is wrong. And it reflects not enough informal English in the training data (or not good enough generalization). We'll fix this in the next release. There isn't any trivial way to fix it at present (except preprocessing the data to pre-tag it or to un-triple the punctuation makrs.

The tag should be . for sentence-final punctuation like . or ! or ?

It was a deliberate decision to keep ??? or !!! as one token. You could argue this, but it makes sense to me, and it is definitely what LDC treebanks do. So, relative to that standard, Google NLP tokenizes wrongly, but that choice then makes it easier to choose the right POS.

**Comment by abdmob** [comment]
Now it is working with '???' (3) but, still not working for '????' (4) in 3.9.2 :(
